core/frontend/vitest.config.mts

```diff
@@ -0,0 +1,68 @@
+import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+export default defineConfig({
+  test: {
+    dir: "src",
+    setupFiles: "./src/test/setupTests.ts",
+    // include: ["**/<testfileName>.test.ts"], // to adhere to chai's describe.only/it.only
+    browser: {
+      provider: "playwright",
+      enabled: true,
+      name: "chromium",
+      headless: true,
+      screenshotFailures: false
+    },
+    coverage: {
+      provider: "v8",
+      include: [
+        "src/**/*"
+      ],
+      exclude: [
+        ...coverageConfigDefaults.exclude,
+        "src/test/**/*",
+        "**/*.d.ts",
+        "**/*.d.tsx"
+      ],
+      reporter: [
```

Comment:
where do these come from? https://vitest.dev/guide/reporters.html

Reply:
These are reporter options for 'istanbul', as for the choices I made, they are based off the reporters property in our tools/build/.nycrc . When using v8, you can still use istanbul reporters, the devs [support](https://github.com/vitest-dev/vitest/blob/main/packages/coverage-v8/src/provider.ts#L4) it